eZ/Publish/Core/Repository/LocationService.php

```diff
@@ -673,7 +673,8 @@ public function moveSubtree(APILocation $location, APILocation $newParentLocatio
                 'new parent Location is in a subtree of the given $location'
             );
         }
-        if (!$newParentLocation->getContent()->getContentType()->isContainer) {
+        $contentTypeId = $newParentLocation->contentInfo->contentTypeId;
+        if (!$this->repository->getContentTypeService()->loadContentType($contentTypeId)->isContainer) {
```

Comment:
Is it intentional change?

Reply:
yes, as Location of ID = 1 does not have ContentInfo, which could provide us with contentTypeId,